GREEDY/Controllers/OCRController.cs

```diff
@@ -11,20 +11,22 @@ namespace GREEDY.Controllers
 {
     public class OCRController
     {
-        private Receipt receipt;
-        private Bitmap image;
+        private Receipt Receipt;
+        private Bitmap Image;
 
         public OCRController (Receipt model, Bitmap image)
         {
-            this.receipt = model;
-            this.image = image;
+            this.Receipt = model;
+            this.Image = image;
         }
 
         public void UseOCR()
         {
             var ocr = new TesseractEngine("../../../Data/tessdata", "eng", EngineMode.TesseractAndCube);
-            var page = ocr.Process(image);
-            receipt.rawText = page.GetText();
+            var page = ocr.Process(Image);
+            //Console.WriteLine(page.GetMeanConfidence());
+            //Receipt.RawText = page.GetText();
```

Comment:
these comments should be deleted in masted file

Reply:
I still wonders if the receipt text should be kept in one big string or each line separatly in a list. Decided on a list, but left the other option commented.